programs/programs/policyengine/calculators/dependencies/member.py

```
@@ -637,6 +638,10 @@ class SsiEarnedIncomeDependency(IncomeDependency):
 class SsiUnearnedIncomeDependency(IncomeDependency):
     field = "ssi_unearned_income"
     income_types = ["unearned"]
+    # Nurturing Futures is only offered as an income source on the CO white label, so
+    # this exclusion is a no-op everywhere else. It reaches SSI, CO ANDCS and CO OAP,
+    # which all read ssi_unearned_income.
+    exclude_income_types = ["nurturingFutures"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)